plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergSessionProperties.java

```diff
@@ -182,6 +183,11 @@ public IcebergSessionProperties(
                         "Parquet: Writer page size",
                         parquetWriterConfig.getPageSize(),
                         false))
+                .add(booleanProperty(
+                    STATISTICS_ENABLED,
+                    "Expose table statistics",
+                    icebergConfig.isTableStatisticsEnabled(),
+                    false))
```

Comment:
i noticed the indentation here is not the same as on then lines above.
you can get formatter definition from https://github.com/airlift/codestyle/

Reply:
yes, I was missing the airflift style file and using something we have internally. Fixed